frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -264,7 +264,7 @@ function MyAlerts(props) {
           <Table className="usa-table usa-table--borderless" fullWidth>
             <caption className="smart-hub--table-caption">
               My activity report alerts
-              <p id="arTblDesc">with sorting</p>
+              <p className="usa-sr-only">with sorting</p>
```

Comment:
`id="arTblDesc"` was being reused, and ids are not supposed to be reused. The id was being used to apply CSS similar to `usa-sr-only`, so we can use that class instead.